simulariumio/data_objects/display_data.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+
+from ..constants import DISPLAY_TYPE
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class DisplayData:
+    name: str
+    radius: float
+    display_type: str
+    url: str
+    color: str
+
+    def __init__(
+        self,
+        name: str,
+        radius: float = None,
+        display_type: str = DISPLAY_TYPE.SPHERE,
+        url: str = None,
+        color: str = None,
+    ):
+        """
+        This object contains info about how to display an agent
+
+        Parameters
+        ----------
+        name : str
+            A string display name for this type of agent
+            Default: use names from simulator data if possible
+        radius : float (optional)
+            A float radius for rendering this agent.
+            For fibers, this is the thickness of the line
+            For default agents, this is the scale of the representation
+            Default : 1.0
```

Comment:
why isn't the default set in `__init__` for this? Is it handled somewhere else?

Reply:
yeah some of the converters use it as an override only if it has more than a default value, otherwise they use the value in the data. If it isn't specified in the data or here, it does default to 1.0 in the converters. I could set it to 1.0 here but it's harder to do the float comparison than a None check.

Reply:
added a comment about this